Fix shebang perl to use env perl instead of hardcoding.

Åke Sandgren, 2022-03-21
diff -ru octopus-11.4.orig/src/fdep/fortran_dependencies.pl octopus-11.4/src/fdep/fortran_dependencies.pl
--- octopus-11.4.orig/src/fdep/fortran_dependencies.pl	2022-01-27 20:22:53.000000000 +0100
+++ octopus-11.4/src/fdep/fortran_dependencies.pl	2022-03-21 07:55:32.654711496 +0100
@@ -1,4 +1,4 @@
-#!/usr/bin/perl -w
+#!/usr/bin/env perl
 #
 # Copyright 2015 Lorenz Hüdepohl
 #
